Applications/prefix_sum/README.md

```diff
@@ -1,44 +1,61 @@
 # Applications: Prefix Sum Example
 
 ## Description
+
 This example showcases a GPU implementation of a prefix sum via a scan algorithm.
 This example does not use the scan or reduce methods from rocPRIM or hipCUB (`hipcub::DeviceScan::ExclusiveScan`) which could provide improved performance.
 
 For each element in the input, prefix sum calculates the sum from the beginning up until the item:
 
 $a_n = \sum^{n}_{m=0} A[m]$
 
-The algorithm used has two phases which are repeated: 
-  A) the block wide prefix sum which uses a two pass prefix sum algorithm as described in _Prefix Sums and Their Applications_ (Blelloch, 1988).
-  B) the device wide prefix sum which propagates values from one block to others.
+The algorithm used has two phases which are repeated:
+
+  a) the block wide prefix sum which uses a two pass prefix sum algorithm as described in _Prefix Sums and Their Applications_ (Blelloch, 1988).
+
+  b) the device wide prefix sum which propagates values from one block to others.
 
-Below is an example where the threads per block is 2. 
-In the first iteration ($\text{offset}=1$) we have 4 threads combining 8 items. 
+Below is an example where the threads per block is 2.
+In the first iteration ($\text{offset}=1$) we have 4 threads combining 8 items.
 
-![](prefix_sum_diagram.svg)
+![prefix_sum_diagram.svg](prefix_sum_diagram.svg)
```

Comment:
The text in between the `[...]` should be a description. There are more of such cases (just search for `![`).